package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "notation",
-    "version": "1.0.0",
+    "version": "1.2.0",
```

Comment:
Can you create an issue to track creating a release checklist?

Reply:
Issue created: https://github.com/notaryproject/notation-action/issues/83